src/Core/ConstraintValidator/TypedRegexValidator.php

```diff
@@ -141,6 +141,8 @@ private function getPattern($type)
                 return '/^[a-zA-Z0-9_-]+$/';
             case TypedRegex::TYPE_URL:
                 return $this->characterCleaner->cleanNonUnicodeSupport('/^[~:#,$%&_=\(\)\.\? \+\-@\/a-zA-Z0-9\pL\pS-]+$/u');
+            case TypedRegex::TYPE_COORDINATE:
+                return '/^\-?[0-9]{1,8}\.[0-9]{1,8}$/s';
```

Comment:
Could you add an unit test `tests/Unit/Core/ConstraintValidator/TypedRegexValidatorTest.php` ?